src/Component.js

```diff
@@ -6,6 +6,7 @@ import copy from 'copy-to-clipboard';
 export class CopyToClipboard extends React.PureComponent {
   static propTypes = {
     text: PropTypes.string.isRequired,
+    onClickCapture: PropTypes.func,
```

Comment:
Still the question is there - what is this onClickCapture prop? Another point - it is not passed thru anymore, so it is a breaking change (major version release)